Automattic/wp-calypso
# Add PlanFeatures (redesigned plans) to the NUX flow

Fix for #6302. Replaces the current plans implementation at the NUX flow plans page with the redesigned plans components.

## Screenshots

### Without Custom Domain

![selection_024](https://cloud.githubusercontent.com/assets/4988512/16458828/dd1eb4f4-3e20-11e6-8011-7cbdfc7de294.jpg)

### With Custom Domain

![selection_022](https://cloud.githubusercontent.com/assets/4988512/16456963/24804ec2-3e1a-11e6-8bc8-61e1842f8eea.jpg)

## Testing Instructions

1. Start Calypso with `ENABLE_FEATURES=manage/plan-features make run`;
2. Navigate to http://calypso.localhost:3000/start/test-plans;
3. Go through the flow with console open;
4. Notice the new redesigned plans page;
5. Does it look alright? Does it look the same as on the `/plans` page?
6. Click on the upgrade btn on each plan;
7. Does it move you to the next step and puts the plan into your cart?
8. Try the above with and without custom domain. It should display four plans without custom domain and three plans with custom domain.

## Todo

- [x] Add Free plan when custom domain is not selected
- [x] Fix visuals of plans header, especially when there are four plans. @gwwar in #6465.

cc @gwwar @artpi @mtias @rralian 

Test live: https://calypso.live/?branch=add/plan-features-nux
